alpha/declcfg/load.go

```diff
@@ -118,7 +118,7 @@ func extractCSV(objs []string) string {
 	return ""
 }
 
-func readYAMLOrJSON(r io.Reader) (*DeclarativeConfig, error) {
+func LoadReader(r io.Reader) (*DeclarativeConfig, error) {
```

Comment:
What was the benefit of the rename?  I may be missing some background but `readYAMLOrJSON`, while verbose, does tell us what it does while `LoadReader` is more vague. 

Reading between the lines ... is it because this interface was not exported?

Reply:
I guess the reason I'd lean towards reader vs yaml or json is just because I feel like I'd be more likely to assume that readYAMLOrJSON would take yaml or json as an input vs a reader, but I'm also not super wedded to it.

Reply:
IMO, its because we're going to export it, and since we're exporting it, we're following the pattern of the other exported functions that return `(declcfg.DeclarativeConfig, error)`